### PR TITLE
Fix long-running operations for VnetGw, NetworkProfile; Fix examples

### PR DESCRIPTION
**Pull request by EvgenyAgafonchikov**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by EvgenyAgafonchikov** [comment]
Spellchecker failures should be fixed after merging this PR: https://github.com/Azure/azure-rest-api-specs/pull/6105

**Comment by dsgouda** [comment]
@shahabhijeet I believe you have a better understanding of the x-ms-long-running-options part, would you consider taking this over

File: specification/network/resource-manager/Microsoft.Network/stable/2016-12-01/examples/RouteFilterCreate.json [hunk 1]
```diff
@@ -22,8 +21,7 @@
               ]
             }
           }
-        ],
-        "peerings": []
```
Comment (sergey-shandar) [inline specification/network/resource-manager/Microsoft.Network/stable/2016-12-01/examples/RouteFilterCreate.json]: Potential API breaking change.

**Review by sergey-shandar — APPROVED** [review]

**Comment by sergey-shandar** [comment]
@number213 @MikhailTryakhov could you approve this?

**Review by KrisBash — APPROVED** [review]
LGTM

**Review by anton-evseev — APPROVED** [review]
LGTM
@EvgenyAgafonchikov please also copy these changes to `2019-06-01` in `network-may-release`

**Pull request merged**